utils/scripts/dbaas_utils.py

```diff
@@ -372,3 +372,35 @@ def create_provision_instance_yaml(isv, prov_acc_ns, deploy_instance_ns):
         data["spec"]["otherInstanceParams"]["projectName"] = instance
     print(data)
     return yaml.dump(data, sort_keys=False)
+
+
+def add_policy_connection_namespaces(namespace: list):
+    """Adding Connection Namespace on DBaaSPolicy"""
+    sl = BuiltIn().get_library_instance("SeleniumLibrary")
+    btn_add_conn_xpath = "//button[@id='root_spec_connectionNamespaces_add-btn']"
```

Comment:
I think we should put this in the resources/object_repo/create_dbaaspolicy_obj.resource. I think it does not look good being here and we should also be consistent. WDYT?

Reply:
The risk of this keyword is not limited to this object but also other dynamic ones on the other parts. I will include this changes, but it still has the risk and it also introduces additional layers to pull the objects from the resource file.

Reply:
I do not understand what you mean. Are you saying that because this object is defined in the low layer, putting it in resources/object_repo/create_dbaaspolicy_obj.resource could cause issues?

Reply:
It wont cause issues, the risk of maintaining the objects due to changes still lies in low level  as we are dynamically updating the objects with index value.